job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/toplevel/LockableResourcesContext.groovy

```diff
@@ -19,4 +20,11 @@ class LockableResourcesContext implements Context {
     void resourceNumber(int resourceNumber) {
         this.resourceNumber = resourceNumber
     }
+
+    /**
+     * Label assigned to group of lockable resources
+     */
+    void labelName(String labelName) {
```

Comment:
add a @since GroovyDoc tag and a @RequiresPlugin annotation specifying a minimum version.